PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/webapp/webappconfig/WebAppRunState.java

```diff
@@ -93,15 +96,14 @@ public WebAppRunState(Project project, WebAppSettingModel webAppSettingModel) {
 
     @Nullable
     @Override
-    public WebApp executeSteps(@NotNull RunProcessHandler processHandler
+    public WebAppBase executeSteps(@NotNull RunProcessHandler processHandler
             , @NotNull Map<String, String> telemetryMap) throws Exception {
         File file = new File(webAppSettingModel.getTargetPath());
         if (!file.exists()) {
             throw new FileNotFoundException(String.format(NO_TARGET_FILE, webAppSettingModel.getTargetPath()));
         }
-        WebApp webApp = getWebAppAccordingToConfiguration(processHandler);
-
-        processHandler.setText(STOP_WEB_APP);
+        WebAppBase webApp = getDeployTargetByConfiguration(processHandler);
```

Comment:
`WebAppBase webApp` the name webApp is somehow confusing, because it might be a WebApp or a DeploymentSlot. If it's a deploymentSlot, it also confuses in following code like `webApp.stop()`.

Reply:
Will rename it to `deployTarget` as the get method indicates.